db/migrate/20211211170003_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :posts do |t|
+      t.string :string
```

Comment:
- [ ] I guess this should be the title with type of string. Make this change. From `:string` to `:title`

Reply:
done